core/src/validator.rs

```diff
@@ -36,7 +36,7 @@ use {
     solana_ledger::{
         bank_forks_utils,
         blockstore::{Blockstore, BlockstoreSignals, CompletedSlotsReceiver, PurgeType},
-        blockstore_db::BlockstoreRecoveryMode,
+        blockstore_db::{BlockstoreOptions, BlockstoreRecoveryMode},
```

Comment:
To me, `blockstore_db` provides the implementation of `blockstore` and it'd be nice if clients could import everything they need from `blockstore`.

What do you think about re-exporting`BlockstoreOptions` like we do some of the other items here:
https://github.com/solana-labs/solana/blob/557d35ec7953eb11db57b7cfe57a9897ebc83540/ledger/src/blockstore.rs#L4

Reply:
Hmm, think I left this comment on the wrong PR; I'd say feel free to ignore it and we can implement my change on all things (AccessType, RecoveryMode, BlockstoreOptions, etc) if we'd like to at a later point